src/main/java/com/criteo/vips/VipsImageImpl.java

```diff
@@ -57,6 +57,21 @@ public VipsImageImpl(VipsImageImpl image, PixelPacket color) throws VipsExceptio
         newFromImage(image, color);
     }
 
+    private VipsImageImpl(int width, int height) throws VipsException {
+        blackNative(width, height);
+    }
```

Comment:
maybe we can just declare a static method

Reply:
done